.github/workflows/examples.yml

```diff
@@ -1,19 +1,19 @@
 name: Examples Integration
 on:
   pull_request:
-    paths-ignore:
-      - "**.md"
-      - "docs/**"
-      - "website/**"
-      - "cdk.tf/**"
-      - "tools/update-github-project-board/**"
+    paths:
```

Comment:
maybe add an info / link here (and in the other workflow files) that notes that changes here also need to be done to the Mergify config so CI does not accidentally block on this (because Mergify waits on success for a job that is newly ignore by changes here).